spec/index.bs

```diff
@@ -556,7 +556,30 @@ dictionary GPULimits {
 A {{GPUBuffer}} represents a block of memory that can be used in GPU operations.
 Data is stored in linear layout, meaning that each byte of the allocation can be
 addressed by its offset from the start of the {{GPUBuffer}}, subject to alignment
-restrictions depending on the operation.
+restrictions depending on the operation. Some {{GPUBuffer}} can be mapped which
```

Comment:
This is the explainer part, I'm thinking the section about map / unmap would have more details, including the explanation of ownership transfer to prevent data races, etc. Would that work for you?